pytorch_blade/scripts/build_pytorch_blade.sh

```diff
@@ -53,7 +53,8 @@ function ci_build() {
       python3 ../scripts/python/common_setup.py --cpu_only $COMMON_SETUP_ARGS
     fi
     TORCH_LIB=$(python -c 'import torch; import os; print(os.path.dirname(os.path.abspath(torch.__file__)) + "/lib/")') \
-    export LD_LIBRARY_PATH=$TORCH_LIB:$LD_LIBRARY_PATH \
+    export LD_LIBRARY_PATH=$TORCH_LIB:$LD_LIBRARY_PATH
+    rm -rf ../tf_community/.tf_configure.bazelrc
```

Comment:
Why is the line required?

Reply:
Sometimes the CI task can not GC the `.tf_configure.bazelrc`, the previous file will cause some unexpected errors.

Reply:
I updated this PR and using `tao_build.py` to setup building env, so I deleted `rm -rf .tf_configure.bazelrc `.